src/iop/overexposed.c

```diff
@@ -185,10 +138,17 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
   const float *const restrict in = __builtin_assume_aligned((const float *const restrict)ivoid, 64);
   float *const restrict out = __builtin_assume_aligned((float *const restrict)ovoid, 64);
 
+  const dt_iop_order_iccprofile_info_t *const current_profile = dt_ioppr_get_pipe_current_profile_info(self, piece->pipe);
+  const dt_iop_order_iccprofile_info_t *const work_profile = dt_ioppr_get_histogram_profile_info_for_overexposed(dev);
+
   // display mask using histogram profile as output
-  _transform_image_colorspace(self, in, img_tmp, roi_out);
+  // FIXME: the histogram already does this work -- use that data instead?
+  if(current_profile && work_profile)
+    dt_ioppr_transform_image_colorspace_rgb(in, img_tmp, roi_out->width, roi_out->height, current_profile,
+                                            work_profile, self->op);
+  else
+    fprintf(stderr, "[_transform_image_colorspace] can't create transform profile\n");
```

Comment:
inside brackets we put the name of the failing function, here it is process in overexpose so I'll use "overexposed.process".

Reply:
Checked the convention from other iop files, seems there is a space instead of dot between iop name and function. But anyway, changed. Thanks.